packages/keystatic/src/component-blocks/cloud-image-preview.tsx

```diff
@@ -281,7 +281,7 @@ function Placeholder(props: {
         paddingX="large"
       >
         <Icon src={imageIcon} />
-        <Text>Cloud image, awaiting configuration…</Text>
+        <Text>Cloud image (click to configure)</Text>
```

Comment:
The `Placeholder` won't render if there's an image source. How could a user ever click this non-interactive element?